### ISSUE TITLE
IP address of the server in icestats

### ISSUE THREAD

**Issue by waster** [karlheyes/icecast-kh#332]
Hello,

Is it possible some how to get listening IP address(es) of the server with xsl from icestats somehow?

We need it for the monitoring purposes of the icecast pool.

**Comment by karlheyes** [comment]
listeners IP addresses are available on the admin/listclients[.xsl] page, but if you are wanting the listening socket address of the server then that is not in the stats but could be made available I suppose.

karl.

**Comment by waster** [comment]
Yes, I mean listening socket address, because it will be very helpful in case of monitoring pool of icecast servers and getting particular IP if all servers are hidden behind one DNS name (weighted DNS pool).
